### DISCUSSION TITLE
Is it possible to use supabase database only on server side in next js...?

### DISCUSSION THREAD

**Discussion by rajatdhoot123** [supabase/supabase#10563] [Questions]
Will be creating a WhatsApp bot using nextjs server APIs and want to store user get from WhatsApp in the database which required no use of a client (browser).

What is the best way to use the database only on the server side?

**Comment by mym1990** [comment]
If you are on Next 13/React 18 you can start using React Server Components(https://nextjs.org/docs/advanced-features/react-18/server-components). 

If that is not an option for you, you can run your API and server logic through the /pages/api folder that comes with a NextJS project load(https://nextjs.org/docs/api-routes/introduction). 

Both of these solutions are server side only.
